lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -139,7 +139,16 @@ def store_column(column_title, format_string, new_data):
             ),
         )
 
-    store_column("minimum_length (-)", "%d", minimum_length)
+    if not (None in minimum_length):
+        store_column("minimum_length (-)", "%d", minimum_length)
+    else:
+        warnings.warn(
+            RuntimeWarning(
+                "This KymotrackGroup has no minimum length. Some analyses may not be possible with "
+                "this file. To get rid of this error and add the metadata, call "
+                "`lk.filter_tracks()` on this file with a given minimum track length."
+            )
+        )
```

Comment:
1. I think you should promote this to an error. `np.savetxt()` will produce an error anyway. It's a bit weird to get a warning and then immediately an error. The warning text will get lost in there.
2. Wording suggestion: You mention a file in the text but the user just has a track group. Maybe say that this error happens for tracks imported from older files that did not contain all metadata.